src/main/java/de/uni_hamburg/informatik/swt/se2/mediathek/services/vormerk/VormerkServiceImpl.java

```diff
@@ -98,4 +108,14 @@ private boolean entferneErstenVormerker(Kunde kunde, Medium medium)
             return false;
         }
     }
+    
+    public void entferneVormerker(Medium medium, Kunde kunde)
+    {
+    	Vormerkkarte vormerkkarte = getVormerkkarte(medium);
+    	if (kunde.equals(vormerkkarte.getErsterVormerker()))
+    	{
+    		vormerkkarte.entferneErstenVormerker();
+    		informiereUeberAenderung();
```

Comment:
Wir wollen das nirgendswo anders außer in entferneErsteVormerker aufrufen, wieso dann public?